### DISCUSSION TITLE
panorama-loaded event trigger one only

### DISCUSSION THREAD

**Discussion by cozybim** [mistic100/Photo-Sphere-Viewer#898] [General]
In the document, panorama-loaded will triggered when a panorama image has been loaded. It mean whenever change the panorama image, it will launch. But I saw that, it only run with the first panorama.

**Comment by mistic100** [comment]
Please use issues to report bugs, not discussions. And this has been fixed yesterday https://github.com/mistic100/Photo-Sphere-Viewer/releases/tag/5.1.3
